hazelcast/src/main/java/com/hazelcast/internal/management/dto/ClientEndPointDTO.java

```diff
@@ -56,11 +56,11 @@ public JsonObject toJson() {
         root.add("address", address);
         root.add("clientType", clientType);
         root.add("name", name);
-        JsonObject attrObject = Json.object();
-        for (Map.Entry<String, String> entry : attributes.entrySet()) {
-            attrObject.add(entry.getKey(), entry.getValue());
+        JsonObject labelsObject = Json.object();
+        for (String label : labels) {
+            labelsObject.add("label", label);
```

Comment:
This serialization is not correct. It leads to duplicate keys in JSON:
```
{
  "labels": {
    "label": "firstLabel-hzclient-1",
    "label": "secondLabel-hzclient-1"
  }
}
```

An array of strings should be used, so the result would be:
```
{
  "labels": [
    "firstLabel-hzclient-1",
    "secondLabel-hzclient-1"
  ]
}
```

Reply:
Good catch

Reply:
Changed